runtime/scripts/question.js

```diff
@@ -876,6 +882,7 @@ Question.prototype = /** @lends Numbas.Question.prototype */
                 q.setCurrentPart(q.getPart(qobj.currentPart));
             }
         });
+        this.events.trigger('resume');
```

Comment:
I would make this a signal called `resumed`, and put it at the end of the function just above